src/image_builder.py

```diff
@@ -50,6 +51,21 @@ def _find_image_object(images_list, image_name):
     return ret_image_object
 
 
+def is_private_artifact_label_required(artifact_uri, image_repo_uri):
```

Comment:
if this function is only required for private repo, lets not add it in public repo